src/js/themes/base.d.ts

```diff
@@ -1079,6 +1079,13 @@ export interface ThemeType {
         color?: ColorType;
         cursor?: string;
       };
+      pinned?: {
```

Comment:
Good catch that this was missing. Thanks for adding.

Reply:
thanks!